jekyll/_cci2/testing-ios.md

```diff
@@ -453,21 +453,18 @@ Environment Variables add two new items named `CRASHLYTICS_API_KEY` and
 
 ### TestFairy
 
-To set up your app on TestFairy follow these steps:
+To set up your app on TestFairy, follow these steps:
 
 ![TestFairy preferences image](  {{ site.baseurl }}/assets/img/docs/testfairy-open-preferences.png)
 
-1. Visit the preferences page in the
-TestFairy dashboard and navigate to the API Key section. 
-2. Copy your API
-key and go to your App's Project settings on CircleCI. 
-3. Add a new
-Environment Variable named `TESTFAIRY_API_KEY` and paste in the API key
-from the TestFairy dashboard.
-4. You can use curl or fastlane for deployment. Here is an example of a curl run command:
+1. On the TestFairy dashboard, navigate to the Preferences page.
+2. On the Preferences page, go to the API Key section.
+3. Copy your API key and go to your application's project settings within the CircleCI application.
+4. To deploy, add a job to your configuration using [fastlane](https://docs.fastlane.tools/getting-started/ios/beta-deployment/) or `curl` (example below).
```

Comment:
I'm not sure about this (you are adding a curl deploy step with a conditional in the example not a job). For 2.0 we want to describe workflows instead of these conditionals for deployment I think.

Reply:
Agreed - you could add a branch filter to the `job` or make a workflow.

Reply:
Good catch! The goal of this PR is to clean up the efforts of the previous PR. That's my fault for not requesting a review and incorporating our best practices re: workflows.

I'll clean this up and be sure to add @marcomorain to future PRs in this area.